css/style.css

```
@@ -1,9 +1,164 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
 * {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
 }
 
 :root {
   --primary: #046C5C;
   --secondary: #0F2F51;
 }
+
+a{
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1250px;
+  margin: auto;
+  font-family: 'Poppins', sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+ul {
+  list-style: none;
+}

```

```suggestion

a{
  text-decoration: none;
}
ul {
  list-style: none;
}

```

Put ui under a and below ui Comment that footer CSS starting from hear. and end with here.

Do this after that good to go

---

Done